src/RewriteConfig.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/*
+ * UrlRewrite Bundle for Contao Open Source CMS.
+ *
+ * @copyright  Copyright (c) 2017, terminal42 gmbh
+ * @author     terminal42 <https://terminal42.ch>
+ * @license    MIT
+ */
+
+namespace Terminal42\UrlRewriteBundle;
+
+class RewriteConfig
```

Comment:
This should get an interface and only the interface should be used in the controllers etc. I guess only the getters are relevant for the interface :)